task/prometheus-gate/0.1/prometheus-gate.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: prometheus-gate
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: gate
+    tekton.dev/displayName: prometheus gate
+spec:
```

Comment:
Could you please add description as well please:
```
 spec:
     description: >-
       One line summary of the task.       👈 # Summary

       Description of the task 
  ```

Reply:
Added thanks!